### ISSUE TITLE
Does voro support gpu acceleration?

### ISSUE THREAD

**Issue by zhoujingyu13687306871** [chr1shr/voro#31]
Dear prof Chris Rycroft：

Thank you very much for developing the voro library. There is a problem that has been bothering me recently. When I use this library in lammps, I find that this library does not seem to support GPU acceleration, and the GPU utilization is always 0. This library can only  used on the CPU ?

**Comment by chr1shr** [comment]
No, Voro++ does not support GPU acceleration. We have discussed this as a possible future direction for development, but it would likely involve a major overhaul of the core routines. It would be a long project.

Recently though we have completed a version of Voro++ that uses multithreading via the OpenMP library. There is a [manuscript currently in review](https://arxiv.org/abs/2209.11606) on this work. The code is available in the `dev` branch of the repository. It has very high parallel efficiency: you can often get close to 100% efficiency for a large system on a multi-core machine.

The `dev` version has been well-tested at this point so you can try it out. But it's currently not integrated with LAMMPS and there are technical issues about how to do that.

**Comment by zhoujingyu13687306871** [comment]
@chr1shr thank you prof Chris Rycroft , I get it
